supertokens_python/recipe/session/claim_base_classes/primitive_claim.py

```diff
@@ -0,0 +1,199 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License") as published by the Apache Software Foundation.
+#
+# You may not use this file except in compliance with the License. You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+from typing import Any, Callable, Dict, Optional, TypeVar, Union
+
+from supertokens_python.types import MaybeAwaitable
+from supertokens_python.utils import get_timestamp_ms
+
+from ..interfaces import JSONObject, JSONPrimitive, SessionClaim, SessionClaimValidator
+
+_T = TypeVar("_T")
+
+
+class HasValueSCV(SessionClaimValidator):
+    def __init__(
+        self, id_: str, claim: SessionClaim[JSONPrimitive], params: Dict[str, Any]
+    ):
+        super().__init__(id_)
+        self.claim: SessionClaim[JSONPrimitive] = claim
+        self.params = params
+
+    def should_refetch(
+        self,
+        payload: JSONObject,
+        user_context: Union[Dict[str, Any], None] = None,
+    ):
+        return self.claim.get_value_from_payload(payload, user_context) is None
+
+    async def validate(
+        self,
+        payload: JSONObject,
+        user_context: Union[Dict[str, Any], None] = None,
+    ):
+        val = self.params["val"]
+        claim_val = self.claim.get_value_from_payload(payload, user_context)
+        is_valid = claim_val == val
+        if is_valid:
+            return {"isValid": True}
+
+        return {
+            "isValid": False,
+            "reason": {
+                "message": "wrong value",
+                "expectedValue": val,
+                "actualValue": claim_val,
+            },
```

Comment:
I prefer if you return a python object where is_valid is a boolean and reason is a JSON object which is optional

Reply:
Okay. TODO

Reply:
Done.

Reply:
Done